AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/test/Mutations/TestHierarchyVersion.dfy

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+include "../../src/Index.dfy"
+include "../AdminFixtures.dfy"
+include "../../../AwsCryptographyKeyStore/test/Fixtures.dfy"
+
+module {:options "/functionSyntax:4" } TestHierarchyVersion {
+  import Types = AwsCryptographyKeyStoreAdminTypes
+  import KeyStoreTypes = AwsCryptographyKeyStoreTypes
+  import KeyStoreErrorMessages
+  import AdminFixtures
+  import Fixtures
+  import opened Wrappers
+
+  method {:test} TestInitializeMutationFailsWithNonUniqueBranchKeyContext() {
+
+    var testId := "DO-NOT-EDIT-Branch-Key-For-HasUniqueTransformedKeys-Check";
+    var ddbClient :- expect Fixtures.ProvideDDBClient();
+    var kmsClient :- expect Fixtures.ProvideKMSClient();
+    // Commented code that adds {"Robbie": "Is a dog."} to the dynamodb item "DO-NOT-EDIT-Branch-Key-For-HasUniqueTransformedKeys-Check" in table KeyStoreDdbTable
+    // This code will create a branch key and make changes so that branch key item contains non unique branch key context key
```

Comment:
This comment on line (21 and 22) is changed in downstream PR https://github.com/aws/aws-cryptographic-material-providers-library/pull/1411/files